Vagrantfile

```diff
@@ -86,23 +86,15 @@ def packages_darwin
     # install all the (security and other) updates
     sudo softwareupdate --ignore iTunesX
     sudo softwareupdate --ignore iTunes
-    sudo softwareupdate --ignore "Install macOS High Sierra"
-    sudo softwareupdate --install --all
-    # get osxfuse 3.x release code from github:
-    curl -s -L https://github.com/osxfuse/osxfuse/releases/download/osxfuse-3.10.4/osxfuse-3.10.4.dmg >osxfuse.dmg
-    MOUNTDIR=$(echo `hdiutil mount osxfuse.dmg | tail -1 | awk '{$1="" ; print $0}'` | xargs -0 echo) \
-    && sudo installer -pkg "${MOUNTDIR}/Extras/FUSE for macOS 3.10.4.pkg" -target /
-    sudo chown -R vagrant /usr/local  # brew must be able to create stuff here
-    which brew || ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    sudo softwareupdate --ignore Safari
+    #sudo softwareupdate --ignore "Install macOS Catalina"
+    #sudo softwareupdate --install --all
+    which brew || CI=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
     brew update > /dev/null
-    brew install pkg-config || brew upgrade pkg-config
-    brew install readline || brew upgrade readline
-    brew install openssl@1.1 || brew upgrade openssl@1.1
-    brew install zstd || brew upgrade zstd
-    brew install lz4 || brew upgrade lz4
-    brew install xz || brew upgrade xz  # required for python lzma module
-    brew install fakeroot || brew upgrade fakeroot
-    brew install git || brew upgrade git
+    brew install pkg-config readline openssl@1.1 zstd lz4 xz fakeroot git
```

Comment:
doesn't this fail in case some of them are already installed and thus the form `brew install x | brew upgrade x` is needed?

Reply:
Not an issue. Just causes a warning, but the rest is installed.

`Warning: git 2.28.0 is already installed and up-to-date`

Reply:
and does "brew install" also upgrade if an old version is installed?

Reply:
You're right, it wouldn't upgrade it. Question is if that's relevant, since the VM would be fresh and the install only runs once. Or how do you usually re-run Vagrant for new releases?

Reply:
Forgot. This line further down will upgrade everything. So the `brew install xx || brew upgrade xx` isn't needed.

`brew upgrade  # upgrade everything`